test/e2e/minimatch/minimatch_test.go

```diff
@@ -194,30 +185,24 @@ func TestMinimatchStartup(t *testing.T) {
 		// 	Type: &pb.FunctionConfig_Rest{
 		// 		Rest: &pb.RestFunctionConfig{
 		// 			Host: mmfHost,
-		// 			Port: mmfGRPCPortInt,
+		// 			Port: mmfHTTPPortInt,
 		// 		},
 		// 	},
 		// },
 	}
 
-	for _, cfg := range cfgs {
-		validateFetchMatchesResult(assert, poolTickets, testProfiles, mm, cfg)
+	for _, fc := range fcs {
+		validateFetchMatchesResult(assert, poolTickets, testProfiles, minimatchTc, fc)
 	}
 }
 
-func validateFetchMatchesResult(assert *assert.Assertions, poolTickets map[string][]string, testProfiles []testProfile, mm *Server, mf *pb.FunctionConfig) {
-	be, err := mm.GetBackendClient()
-	assert.Nil(err)
-	assert.NotNil(be)
-
-	mfclose, err := serveMatchFunction()
-	defer mfclose()
-	assert.Nil(err)
+func validateFetchMatchesResult(assert *assert.Assertions, poolTickets map[string][]string, testProfiles []testProfile, tc *rpcTesting.TestContext, fc *pb.FunctionConfig) {
```

Comment:
could use better naming -  testFetchMatches ? validateFetchMatchesResult implies you already have the result and you are validating it - a common pattern used for that case - in this specific case you are actually fetching the results in this function too.